docs/upgrading-openshift.md

```diff
@@ -13,6 +13,34 @@ Occasionally, changes are made to OpenShift Butane configs (those that specify `
 1. TOC
 {:toc}
 
+## From Version 4.18.0 to 4.19.0
+
+There are no breaking changes between versions 4.18.0 and 4.19.0 of the `openshift` configuration specification. Any valid 4.18.0 configuration can be updated to a 4.19.0 configuration by changing the version string in the config.
+
+### Luks cex
+
+The `luks` section gained a new `cex` field for s390x. If specified, the LUKS volume will issue volume encryption using IBM Crypto Express Card.
+
+<!-- butane-config -->
+```yaml
+variant: openshift
+version: 4.19.0
+metadata:
+  name: minimal-config
+  labels:
+    machineconfiguration.openshift.io/role: worker
+openshift:
+  kernel_arguments:
+    - rd.luks.key=/etc/luks/cex.key
+storage:
+  luks:
+    - name: "luks-cex"
+      device: "/dev/sdb"
+      discard: true
+      cex:
+        enabled: true
+```
+
```

Comment:
We should also likely add the section about the new boot layouts here: https://github.com/coreos/butane/blob/main/docs/upgrading-fcos.md#boot_device-layouts-s390x-support